examples/example-gateway/endpoints/tchannel/baz/baz_call_test.go

```diff
@@ -45,6 +45,14 @@ func TestBazCall(t *testing.T) {
 	success, resHeaders, err := ms.MakeTChannelRequest(
 		ctx, "SimpleService", "Call", reqHeaders, args, &result,
 	)
+	dynamicRespHeaders := []string{
+		"client.response.duration",
```

Comment:
Since this is a dynamically generated header key/val, we exclude this from unit tests.